controllers/admin/AdminThemesController.php

```diff
@@ -78,7 +78,7 @@ public function init()
 
     public function downloadAddonsThemes()
     {
-        if (!$this->logged_on_addons) {
+        if (!$this->logged_on_addons || (!in_array($this->authorizationLevel(), array(AdminControllerCore::LEVEL_ADD, AdminControllerCore::LEVEL_EDIT, AdminControllerCore::LEVEL_DELETE) )) || _PS_MODE_DEMO_) {
```

Comment:
120 chars max in PSR :)

So:

```php
if (
    !$this->logged_on_addons
    || (!in_array(
        $this->authorizationLevel(),
        array(AdminControllerCore::LEVEL_ADD, AdminControllerCore::LEVEL_EDIT, AdminControllerCore::LEVEL_DELETE))
    )
    || _PS_MODE_DEMO_
) {
```

And maybe this example is still longer than 120 chars ^^